README.md

```diff
@@ -20,6 +20,45 @@ $ make VERSION=11.0  # generate image camptocamp/odoo-project:11.0-latest
 $ make VERSION=10.0  # generate image camptocamp/odoo-project:10.0-latest
 $ make VERSION=9.0  # generate image camptocamp/odoo-project:9.0-latest
 ```
+## Image version
+
+There is 4 versions of image:
```

Comment:
s/is 4 versions of image/are 4 flavors of the image/

Reply:
done